sql/aggregate/cdta.sql

```diff
@@ -34,7 +33,6 @@ FROM YEARLY_devdb_{{ decade }}
 -- {% endif %}
 
 GROUP BY
-    YEARLY_devdb_{{ decade }}.boro,
```

Comment:
We don't need to group by boro because each cdta is associated with exactly one borough?

Reply:
Actually the opposite. Since some of those cdta is cross boro so we are taking them out to avoid confusion of two records for the same cdta

Reply:
cool